src/components/SettingsDialog.vue

```diff
@@ -218,20 +338,26 @@ export default {
          */
         async setup() {
             try {
-                const response = await api.config.get()
-                const config = response.data
+                await this.$store.dispatch("refreshConfig")
+                const { config } = this.$store.state
                 this.resetPrintImage = false
+                this.resetVisibleInfoBlocks = false
 
                 if (!config) {
                     throw new Error()
                 }
 
                 this.printImage = config.print_image
+                this.visibleInfoBlocks = visibleInfoBlocksDecode(
+                    config.visibleInfoBlocks
+                )
                 this.showTagCloudInRecipeList =
                     this.$store.state.localSettings.showTagCloudInRecipeList
                 this.updateInterval = config.update_interval
                 this.recipeFolder = config.folder
-            } catch {
+            } catch (err) {
+                // eslint-disable-next-line no-console
+                console.error("Error setting up SettingsDialog", err)
```

Comment:
Is this not better done using our logging mechanism?

Reply:
I guess I wanted to be consistent with the rest of the file. Shall I convert them all to use the logging or only this instance?

Reply:
As you like. I would at least fix lines that are changed anyways.